src/test/specs/API-v3/05 - EE-6347 Total Monthly Payments for Salaried Assessments.feature

```diff
@@ -0,0 +1,74 @@
+Feature: Category A & B Financial Requirement - Solo & Combined Applications for Monthly Salary Assessments
+
+    # JIRA STORIES: EE-6347 (Total Monthly Payments for Monthly Salary Assessments)
+    # Scenarios where Categories A & B Monthly Salary Assessments pass
+    # Multiple payments within the same month can be added together and used before the comparison against the annual check takes place
+    # Employment check threshold is £1550 within the immediate 32 days from and inclusive of application raised date
```

Comment:
Only applicable to Category B assessments

Reply:
👍